docs/build/build-storage.md

```diff
@@ -37,49 +36,6 @@ hurdles of the HTTP-based Internet.
 | accessibility: data is distributed to multiple nodes and can be accessed at any time. Bandwidth is high: nearest peer can serve the data | accessibility: data can only be accessed if the server is live and there are no interruptions in transmission. Bandwidth is limited: clients send requests to the same server |
 
-### Filecoin
-
-Like IPFS, [Filecoin](https://filecoin.io/) is a protocol developed by
-[Protocol Labs](https://protocol.ai/) that offers a decentralized storage network. Filecoin's main
-focus is the storage itself and uses IPFS as a [complementary] back-end protocol.
-
-## Substrate Storage
-
-Substrate takes a layered approach to storage by using a key-value data store that is implemented as
-a database-backed, modified Merkle tree. Substrate's higher-layer storage abstractions are built on
-the key-value store.
-
-The key-value data store is backed by [RocksDB](https://rocksdb.org/), and it also supports an
-experimental [Parity database](https://github.com/paritytech/parity-db).
-
-:::note
-
-The database is used for components that require persistent storage These are components like
-Substrate clients, Substrate light-clients & off-chain workers. For more information, check out
-Substrate documentation [Storage Page](https://docs.substrate.io/main-docs/build/runtime-storage/).
-
-:::
-
-When building on Substrate, runtime developers can utilize of Substrate's FRAME `Storage pallet`
-which gives access to Substrate's storage APIs. These storage items support values that are encoded
-by Parity's `SCALE (Simple Concatenated Aggregate Little-Endian) Codec`.
-
-There is a
-[`Storage Value`](https://paritytech.github.io/substrate/master/frame_support/storage/trait.StorageValue.html)
-API that is used to store single values, a
-[`Storage Map`](https://paritytech.github.io/substrate/master/frame_support/storage/trait.StorageMap.html)
-API that is used to a key-value hash map, a
-[`Storage Double Map`](https://paritytech.github.io/substrate/master/frame_support/storage/trait.StorageDoubleMap.html)
-API that creates a `storage map` with two keys to provide the ability to efficiently remove all
-entries that have a common first key, and a
-[`Storage N Map`](https://paritytech.github.io/substrate/master/frame_support/storage/trait.StorageNMap.html)
-API that can be used to store a hash map with any arbitrary number of keys.
-
-These layered APIs act as runtime storage that allows you to store data in your blockchain. More
-information can be found at the
-[`Runtime Storage Page`](https://docs.substrate.io/main-docs/build/runtime-storage/) on Substrate
-Developer Hub.
-
 ### Crust Storage
 
 [Crust Network](https://crust.network) provides a Web3.0 decentralized storage network for the
```

Comment:
Unable to edit this in the review but line 47 has a typo:

"Crust provides a native cross-chain <del>communication</del> communication pallet based on"